build.gradle

```diff
@@ -330,6 +331,13 @@ allprojects {
         i++;
         folder.output = "build-eclipse/" + i
       }
+
+      // Workaround for https://github.com/gradle/gradle/issues/14932
```

Comment:
Could you explain why this works in a comment rather than linking to the issue? It is just easier to read if the explanation is embedded.

Reply:
done